packages/text-editor/src/components/CollaboratorEditor.svelte

```diff
@@ -161,6 +161,25 @@
     })
   }
 
+  export function selectRange (from: number, to: number) {
```

Comment:
Not for this PR, but I think that we need to expose selectRange and selectNode as editor commands, not just exported functions in component.